core/domain/src/test/kotlin/com/google/samples/apps/nowinandroid/core/domain/GetRecentSearchQueriesUseCaseTest.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid.core.domain
+
+import com.google.samples.apps.nowinandroid.core.testing.repository.TestRecentSearchRepository
+import com.google.samples.apps.nowinandroid.core.testing.util.MainDispatcherRule
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.test.advanceUntilIdle
+import kotlinx.coroutines.test.runTest
+import org.junit.Rule
+import org.junit.Test
+import kotlin.test.assertEquals
+
+/**
+ * Unit test for [GetRecentSearchQueriesUseCase]
+ */
+class GetRecentSearchQueriesUseCaseTest {
+
+    @get:Rule
+    val mainDispatcherRule = MainDispatcherRule()
+
+    private val recentSearchRepository = TestRecentSearchRepository()
+
+    private val useCase = GetRecentSearchQueriesUseCase(
+        recentSearchRepository,
+    )
+
+    @Test
+    fun whenQueryLimitIsBy10_recentSearchQueriesAreReturnedUpTo10() = runTest {
+        // Obtain a stream of recent search queries with no param.
```

Comment:
nit: I feel like all these comments are unnecessary, and only explain what the code does with more words than the code itself.

Reply:
@SimonMarquis yes, i completely agree with you. However, to maintain  consistent with already existed unit test class such as 'GetFollowableTopicsUseCase', 'UserNewsResourceTest', 'OfflineFirstNewsRepositoryTest', 'OfflineFirstTopicsRepositoryTest' etc..., i wrote comments like that.

so, do you think that is it better for deleting existing all comment also including my newly PR code comment? or are there any standards for writing comments?